spring-kafka-docs/src/main/antora/modules/ROOT/pages/whats-new.adoc

```diff
@@ -76,3 +76,10 @@ For details, see xref:kafka/receiving-messages/rebalance-listeners.adoc#new-reba
 
 The `DefaultKafkaHeaderMapper` and `SimpleKafkaHeaderMapper` support multi-value header mapping for Kafka records.
 More details are available in xref:kafka/headers.adoc#multi-value-header[Support multi-value header mapping].
+
+[[x40-batch-observability]]
+=== Per-Record Observation in Batch Listeners
+
+It is now possible to get an observation for each record when using a batch listener.
+Previously, only a single observation was created for the entire batch.
```

Comment:
I don't think we ever supported observability for batch listeners, right?

Reply:
you're right, sorry